libs/cardano-ledger-core/testlib/Test/Cardano/Ledger/Common.hs

```diff
@@ -0,0 +1,30 @@
+module Test.Cardano.Ledger.Common
+  ( module X,
+    ledgerTestMain,
+    ledgerTestMainWith,
+    ledgerHspecConfig,
+  )
+where
+
+import Control.Monad as X (forM_, unless, when)
+import System.IO (BufferMode (LineBuffering), hSetBuffering, hSetEncoding, stdout, utf8)
+import Test.Hspec as X
+import Test.Hspec.QuickCheck as X
+import Test.Hspec.Runner
+import Test.QuickCheck as X
+
+ledgerHspecConfig :: Config
+ledgerHspecConfig =
+  defaultConfig
+    { configUnicodeMode = UnicodeNever,
+      configTimes = True
+    }
+
+ledgerTestMainWith :: Config -> Spec -> IO ()
+ledgerTestMainWith conf spec = do
+  hSetBuffering stdout LineBuffering
+  hSetEncoding stdout utf8
+  hspecWith conf spec
+
+ledgerTestMain :: Spec -> IO ()
```

Comment:
I forgot what you said about this, I'm sorry!   Who will it be re-used by, so that it's called Common? 
So far it seems only used from the Main of cardano-ledger-core test suite, but it's generic enough to be used from where else for example?

Reply:
Oh, judging on [this](https://github.com/input-output-hk/cardano-ledger/pull/3218/commits/0929d563f72d19fa3dfa552110736dcced6935a8#diff-3766dcbff475881d6bfc018ec82386fcb978ac2c3cfe0c46bc144c6a9f581c5aR8) import , this is  meant like a little prelude, grouping useful imports and stuff I guess?

Reply:
> Oh, judging by [this](https://github.com/input-output-hk/cardano-ledger/pull/3218/commits/0929d563f72d19fa3dfa552110736dcced6935a8#diff-3766dcbff475881d6bfc018ec82386fcb978ac2c3cfe0c46bc144c6a9f581c5aR8) import , this is meant like a little prelude, grouping useful imports and stuff I guess?

That's exactly right.

We can avoid adding these in every test module:
```haskell
import Test.Hspec
import Test.QuickCheck
...
```